### PR TITLE
[FrameworkBundle] Enable 'debug:translation' command to inspect directories

### PR DESCRIPTION
**Pull request by xelaris**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | yes |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | #12273 |
| License | MIT |
| Doc PR |  |

This enables not only to inspect Bundles, but also directories, like the `app/` directory for instance.

I'm not sure about the branch, strictly speaking it's a new feature, but in some way a bug fix too.

### CODE REVIEW

**Comment by fabpot** [comment]
It adds a new flag, so this is definitely a new feature and it will be merged in 2.7 instead.

File: src/Symfony/Bundle/FrameworkBundle/Command/TranslationDebugCommand.php [hunk 1]
```diff
@@ -45,6 +45,7 @@ protected function configure()
                 new InputOption('domain', null, InputOption::VALUE_OPTIONAL, 'The messages domain'),
                 new InputOption('only-missing', null, InputOption::VALUE_NONE, 'Displays only missing messages'),
                 new InputOption('only-unused', null, InputOption::VALUE_NONE, 'Displays only unused messages'),
+                new InputOption('path', 'p', InputOption::VALUE_NONE, 'Treats bundle name as path'),
```
Comment (fabpot) [inline src/Symfony/Bundle/FrameworkBundle/Command/TranslationDebugCommand.php]: Do we really need a new option? What about looking for a path if the bundle does not exist instead?

**Comment by stof** [comment]
``translation:update`` should also be updated for consistency

**Comment by fabpot** [comment]
@xelaris Opening a new one for 2.7 and make both commands behave the same looks like the way to go. I would not create an abstract class though. Thanks.

**Comment by xelaris** [comment]
closed in favor of #13443

**Pull request closed without merging**